wp-content/plugins/core/src/Theme/Resources/Editor_Styles.php

```diff
@@ -6,24 +6,23 @@
 
 class Editor_Styles {
 	/**
-	 * Visual Editor Styles
-	 * @action after_setup_theme
+	 * Register styles with the block editor.
+	 *
+	 * Uses the relative path, because the block editor will actually grab
+	 * the file from the theme directory and apply some changes to it.
+	 *
+	 * @see https://developer.wordpress.org/block-editor/developers/themes/theme-support/#editor-styles
+	 *
+	 * @action admin_init
 	 */
-	public function visual_editor_styles() {
-
-		$css_dir    = trailingslashit( get_stylesheet_directory_uri() ) . 'assets/css/dist/admin/';
-		$editor_css = 'editor-style.css';
-
-		// Production
-		if ( ! defined( 'SCRIPT_DEBUG' ) || SCRIPT_DEBUG === false ) {
-			$editor_css = 'editor-style.min.css';
-		}
-
-		add_editor_style( $css_dir . $editor_css );
-
+	public function block_editor_styles() {
+		add_editor_style( $this->theme_stylesheet_path() );
+		add_theme_support( 'editor-styles' );
 	}
+
 	/**
-	 * Visual Editor Body Class
+	 * Add a body class to the visual editor
+	 *
 	 * @filter tiny_mce_before_init
 	 */
 	public function visual_editor_body_class( $settings ) {
```

Comment:
Is 'visual_editor' still the right name for these? mce_editor instead maybe? Let me know whatever you think is right, and then i will sync the css.

Reply:
I think `mce-editor` is the right name. I didn't want to change it here until you changed the stylesheet file name. But I guess it doesn't really matter; I'll go ahead with the change for the function names, at least.

Reply:
ok! I will modify that when i do your block blacklist work, which i could get to tonight actually after you merge.